editor/script/editor.js

```diff
@@ -887,10 +886,11 @@ function start() {
 	}
 
 	// localization
-	if (urlFlags["lang"] != null) {
-		localStorage.editor_language = urlFlags["lang"]; // need to verify this is real language?
-	}
-	localization = new Localization();
+	localization = new Localization(urlFlags["lang"]);
+	Store.init(function () {
+		// TODO: localize
+		window.alert('A storage error occurred: The editor will continue to work, but data may not be saved/loaded. Make sure to export a local copy after making changes, or your gamedata may be lost!');
```

Comment:
this should be localized, but i don't know the authoring flow for new locale entries

Reply:
thanks for the note! I'll add a string for that :)